pmd-visualforce/src/main/resources/category/vf/security.xml

```diff
@@ -26,6 +26,29 @@ Avoid calling VF action upon page load as the action becomes vulnerable to CSRF.
         </example>
     </rule>
 
+    <rule name="VfHtmlStyleTagXss"
+          language="vf"
+          since="6.29.0"
+          message="Use correct encoding for expressions within Style tag"
+          class="net.sourceforge.pmd.lang.vf.rule.security.VfHtmlStyleTagXssRule"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_vf_security.html#vfhtmlstyletagxss">
+        <description>
+            Use relevant encoding with EL in html tags
+        </description>
+        <priority>3</priority>
+        <example>
+            <![CDATA[
+<apex:page>
+<style>
+ div {
+  background: url('{!XSSHere}');
```

Comment:
I'll expand the example to provide also a "good" usage here:

`background: url('{!JSINHTMLENCODE(XSSHere)}');`